CRW-EthOnline-smart-contract/contracts/CRWCP.sol

```diff
@@ -1,15 +1,11 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.17;
 
-// any specific tokenomics? max supply? only certain restaurants can mint?
 // what kind of validation do we need to put on the verify function? restaurants can't assign to themselves; whitelist to who the token can be sent
```

Comment:
the restaurants could be whitelisted in advance since each of them would be curated and added to each city campaign. 
in the future we may need an automated way for restaurants to sign up.
for now the foodies need to be able to sign up seamlessly and the restaurants could be whitelisted

Reply:
already implemented